cmd/podman/farm/list.go

```diff
@@ -43,7 +43,7 @@ func init() {
 	flags := lsCommand.Flags()
 
 	formatFlagName := "format"
-	flags.StringVar(&lsOpts.Format, formatFlagName, "", "Format farm output using Go template")
+	flags.StringVarP(&lsOpts.Format, formatFlagName, "f", "", "Format farm output using Go template")
```

Comment:
Why are you changing this? IMO we should not use shorthand `f` for `format`, the other list command use it already with `--filter` so this would violate users expectations and make the cli inconsistent. And it is possible that we want to add a `--filter` here at some point I guess.

Reply:
Yes no short-hand of "f". i have been fooling with a PR to move the --farm to podman farm build only.

Reply:
Ack. I did so for consistency with the already-checked-in man page, a decision that could've obviously gone either way. I chose to honor the man page because we actually don't have any convention for `-f`:
```console
$ make -C docs clean         ! to eliminate false dups between *.md.in and *.md
$ cd docs/source/markdown
$  grep '^####.*--filter.*-f' *|wc -l
13
$ grep '^####.*--format.*-f' *|wc -l
15
```
...plus 17 `--force` and a handful of `--file/--filename`.

Good catch, cross-checking against other `list` commands. I should have been more careful there.

I will push on Monday, because I'd like to do a more thorough job of re-checking my work.

Reply:
Fixed, and re-reviewed to my satisfaction.